tck/src/main/java/org/eclipse/microprofile/lra/tck/TckTests.java

```diff
@@ -208,6 +214,52 @@ public void join() throws WebApplicationException {
                 lraClient.isLRAFinished(lra));
     }
 
+    /**
+     * TCK test to verify that methods annotated with {@link AfterLRA}
+     * are notified correctly when an LRA terminates
+     */
+    @Test
+    public void testAfterLRAParticipant() throws WebApplicationException, InterruptedException {
+        URI lra = lraClient.startLRA(null, lraClientId(), lraTimeout(), ChronoUnit.MILLIS);
+        WebTarget resourcePath = tckSuiteTarget.path(AFTER_LRA_PARTICIPANT_PATH).path(WORK);
+        Response response = resourcePath
+                .request().header(LRA_HTTP_CONTEXT_HEADER, lra).put(Entity.text(""));
+        checkStatusAndCloseResponse(Response.Status.OK, response, resourcePath);
+        lraClient.closeLRA(lra);
+
+        // verify that the LRA is now in one of the terminal states
+        assertTrue("testAfterLRAParticipant: LRA did not finish",
+                lraClient.isLRAFinished(lra, lraMetricService, AfterLRAParticipant.class.getName()));
+
+        // verify that the resource was notified of the final state of the LRA
+        assertEquals("testAfterLRAParticipant: end synchronization was not invoked on resource " + resourcePath.getUri(),
+                1,
+                lraMetricService.getMetric(LRAMetricType.Closed, lra, AfterLRAParticipant.class.getName()));
+    }
+
+    /**
+     * TCK test to verify that methods annotated with {@link AfterLRA}
+     * are notified correctly when an LRA terminates
+     */
+    @Test
+    public void testAfterLRAListener() throws WebApplicationException, InterruptedException {
+        URI lra = lraClient.startLRA(null, lraClientId(), lraTimeout(), ChronoUnit.MILLIS);
+        WebTarget resourcePath = tckSuiteTarget.path(AFTER_LRA_LISTENER_PATH).path(WORK);
```

Comment:
Within testAfterLRAParticipant() and testAfterLRAListener() a reference to AfterLRAParticipant.WORK is made (although there is also a AfterLRAListener.WORK)
Both values are the same but can cause some confusing, issues in the future.